src/screens/Security.js

```diff
@@ -74,11 +74,16 @@ export class Security extends React.Component {
         this.onSafeBiometryDisabled();
       }
     } else {
-      this.setState({ biometryEnabled: value });
-      setBiometryEnabled(value);
+      this.onOldBiometryEnabled();
```

Comment:
Shouldn't we call it only `if value === true`?

And on disable shouldn't we call 

```
this.setState({ biometryEnabled: false });
    setBiometryEnabled(false);
```